style.css

```diff
@@ -278,3 +282,205 @@ footer {
   gap: 20px;
   font-size: 22px;
 }
+
+.hide-mobile {
+  display: none;
+}
+
+.hide-desktop {
+  display: block;
+}
+
+.fa-envelope {
+  display: none !important;
+}
+
+@media screen and (min-width: 768px) {
+  .fa-bars {
+    display: none !important;
+  }
+
+  .fa-envelope {
+    display: block !important;
```

Comment:
- [ ] As much as possible, do not use `!important` with your styles, using `!important` in your CSS is generally considered to be bad practice because it overrides all other styles and can make your stylesheets difficult to maintain